subprojects/testing-native/src/integTest/groovy/org/gradle/nativeplatform/test/xctest/SwiftXCTestIntegrationTest.groovy

```diff
@@ -187,9 +180,9 @@ apply plugin: 'swift-library'
     // TODO: Need to support test report for test case assertion
     @Requires(TestPrecondition.MAC_OS_X)
     def "does not execute removed test suite and case"() {
-        def testBundle = new IncrementalSwiftXCTestRemoveDiscoveryBundle()
-
         given:
+        def testBundle = new IncrementalSwiftXCTestRemoveDiscoveryBundle()
+        settingsFile << "rootProject.name = 'app'"
```

Comment:
This should probably `${testBundle.projectName}`.

Reply:
The incremental fixtures don't have a projectName yet